test/lda.jl

```diff
@@ -71,8 +72,12 @@ import Random
     @test f.w ≈ w_gt
     @test f.b ≈ b_gt
 
+    covestimator = SimpleCovariance(corrected=true)
+    fs = fit(LinearDiscriminant, Xp, Xn; covestimator=covestimator)
+    @test fs.w ≈ w_gt
```

Comment:
Here too, is there a way to check that `covestimator` is used?

Reply:
It's a bit trickier here, both corrected and uncorrected estimators give the same fitted discriminant. I could use a different estimator for testing, even a fake one (for example one that always returns a unit matrix).

Reply:
Ah. If that's not too hard, then why not.

Reply:
OK, done.